tests/opnReqTest

```diff
@@ -125,7 +125,7 @@ build_opnReqTests() {
       ecflow_create_compile_task
     else
       echo "compiling $name with compile option $MAKE_OPT"
-      ./compile.sh $MACHINE_ID "${MAKE_OPT}" $name >${LOG_DIR}/compile_${TEST_NAME}_$name.log 2>&1
+      ./compile.sh $MACHINE_ID "${MAKE_OPT}" $name >${LOG_DIR}/compile_${TEST_NAME}_$name.log #2>&1
```

Comment:
same here. I think better to revert to original

Reply:
@jkbk2004 Also reverted